go.mod

```diff
@@ -290,6 +290,10 @@ require github.com/grafana/pyroscope/api v0.2.1 // @grafana/observability-traces
 
 require github.com/apache/arrow/go/v13 v13.0.0 // @grafana/observability-metrics
 
+require github.com/oklog/ulid/v2 v2.1.0 // @grafana/grafana-app-platform-squad
```

Comment:
I think this can be removed now.

```
git checkout origin/main -- go.mod
git checkout origin/main -- go.sum
```

Reply:
go mod tidy seems to think it's still needed

Reply:
Ahh i see:
```
current.Guid = ulid.Make().String()
```
I think that should be a normal UUID.

once removed, then we don't need the ulid import anymore